sdk/ai/azure-ai-generative/README.md

```diff
@@ -1,33 +1,61 @@
-# Azure Template Package client library for Python
-
-This template package matches necessary patterns that the development team has established to create a unified SDK. The packages contained herein can be installed singly or as part of the `azure` namespace. Any other introductory text should go here.
+# Microsoft Azure SDK for Python
 
+This is the Microsoft Azure MyService Management Client Library.
 This package has been tested with Python 3.7+.
+For a more complete view of Azure libraries, see the [azure sdk python release](https://aka.ms/azsdk/python/all).
+
+## _Disclaimer_
+
+_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
+
+## Getting started
+
+### Prerequisites
+
+- Python 3.7+ is required to use this package.
+- [Azure subscription](https://azure.microsoft.com/free/)
+
+### Install the package
+
+```bash
+pip install azure-ai-generative
+pip install azure-identity
+```
 
-For a more complete set of Azure libraries, see https://aka.ms/azsdk/python/all.
+### Authentication
 
-# Getting started
+By default, [Azure Active Directory](https://aka.ms/awps/aad) token authentication depends on correct configure of following environment variables.
 
-For a rich example of a well formatted readme, please check [here.](https://github.com/Azure/azure-sdk/blob/main/docs/policies/README-TEMPLATE.md) In addition, this is an [example readme](https://github.com/Azure/azure-sdk/blob/main/docs/policies/README-EXAMPLE.md) that should be emulated. Note that the top-level sections in this template align with that of the [template.](https://github.com/Azure/azure-sdk/blob/main/docs/policies/README-TEMPLATE.md)
+- `AZURE_CLIENT_ID` for Azure client ID.
+- `AZURE_TENANT_ID` for Azure tenant ID.
+- `AZURE_CLIENT_SECRET` for Azure client secret.
 
-# Key concepts
+In addition, Azure subscription ID can be configured via environment variable `AZURE_SUBSCRIPTION_ID`.
 
-Bullet point list of your library's main concepts.
+With above configuration, client can be authenticated by following code:
 
-# Examples
+```python
+from azure.identity import DefaultAzureCredential
+from azure.ai.generative import 
+import os
```

Comment:
nit: maybe move this line to the top?

Reply:
This is also the auto generated code from the CI bot, the bot is disabled now.